source/daplink/interface/swd_host.c

```diff
@@ -1005,10 +1005,19 @@ uint8_t swd_set_target_state_sw(TARGET_RESET_STATE state)
             }
 
             // Perform a soft reset
-            if (!swd_write_word(NVIC_AIRCR, VECTKEY | SOFT_RESET)) {
+#if (__CORTEX_M >= 0x03)
```

Comment:
This define is for the MCU running DAPLink when really it depends on the target MCU. Could you define ```SCB_AIRCR_PRIGROUP_Msk``` at the top of this file so it is present for all MCUs?